sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala

```diff
@@ -404,6 +404,9 @@ class SQLQuerySuite extends QueryTest with SharedSparkSession with AdaptiveSpark
     checkAnswer(
       sql("SELECT COALESCE(null, null, null)"),
       Row(null))
+    checkAnswer(
+      sql("SELECT COALESCE(s, n) FROM nullStrings"),
```

Comment:
I think this case is covered at least at `sql/core/src/test/resources/sql-tests/inputs/outer-join.sql`

Reply:
WDYT about the above @HyukjinKwon 's comment, @zengruios ? Is this PR different because this is `String`? If this is already covered, shall we close this PR?